### ISSUE TITLE
Eager load a computed property that's using a navigation property

### ISSUE THREAD

**Issue by benmccallum** [hazzik/DelegateDecompiler#159]
I'm not sure if this is even possible, but I'm basically have an entity `Booking` that has a 1:many with `BookingSupplierResponse` and I want to have a `[Computed]` field that plucks out one of these responses. I'm on EF6. 

There's more real-world context in my [SO post](https://stackoverflow.com/questions/60191436/1many-and-11-given-composite-key-in-entity-framework-core-or-6).

```
public class Booking
{
    public int BookingId { get; set; }

    public int SupplierId { get; set; }

    public virtual ICollection<BookingSupplierResponse> SupplierResponses { get; set; }

    [Computed]
    public BookingSupplierResponse SupplierResponse
        => SupplierResponses.SingleOrDefault(sr => sr.SupplierId == this.SupplierId);
}

public class BookingSupplierResponse
{
    public int BookingSupplierResponseId { get; set; }

    public int BookingId { get; set; }
    public virtual Booking { get; set; }

    public int SupplierId { get; set; }
    public virtual Supplier { get; set; }

    // ...
}
```

Problem is that I'm getting a `The ObjectContext instance has been disposed and can no longer be used for operations that require a connection.`, since I've already disposed the DbContext.
I assume that happens because I'm not eager loading the  navigation property, but I don't really want to as I was hoping the `.Decompile()` would figure this out for me. 

I tried doing a `Include(b => b.SupplierResponse)` but then EF complains that there's no navigation property configured for `SupplierResponse`. 

I feel like I could probably do `.Include(b => b.SupplierResponses)` and have it work, but then I'd be blowing out my resultset with a join. 

Any suggestions? Maybe I'll just have to resign myself to another DB query for this.

**Comment by benmccallum** [comment]
Hey @hazzik, in some places another DB query here is fine, specifically where I'm just loading one Booking, but I've found myself needing to load multiple Booking records for a listing and now I'm kinda stuck here. 

Have you had a chance to look at this? Or maybe @jbogard, have you run it a similar situation? I've tried with DelegateCompiler (computed prop on the entity itself) and with AutoMapper (property on a DTO doing `src.SupplierResponses.FirstOrDefault(sr => sr.SupplierId == src.SupplierId)` with no luck.
